Fixed README typo, and Python/Bash confusion

noah-empire opened the pull request:
Fixed typo and updated outdated references to Python dependencies.

noah-empire committed:
Fixed README typo

noah-empire committed:
Python no longer needed?

It looks like you changed the script around so it no longer requires Python and just runs in Bash.  The readme wasn't updated to reflect this and left me confused for a while so hopefully this saves the next person some time.

eddiewebb reviewed (changes requested):
Hey thanks! I think the change removed all dependencies on Python actually. It's all just shell script now.

Would you please update line 36 to reference 'jq'instead, and remove python as executor from line 49?

noah-empire committed:
Fixed executing the `.sh` file with python

Not quite sure what you meant by " please update line 36 to reference 'jq'instead" though.  Can you provide some more details? Thanks!
